### PR TITLE
#121 new homepage

### PR DESCRIPTION
**Pull request by jk-george**
This solves: [Issue 121](https://github.com/users/gem09lo/projects/1?pane=issue&itemId=90530939&issue=gem09lo%7CPriceSlashTrack%7C121)

## Outcome:

We want the best user experience possible. By having a welcome page to our dashboard, we hope to enhance the user experience and have a more appealing look to our service.

New "home page" created for the dashboard, which can navigate to "login" with ease through the sidebar.

### CODE REVIEW

File: streamlit_dashboard/.streamlit/config.toml
Comment (indiah444) [inline streamlit_dashboard/.streamlit/config.toml:1]: Great theme, links well to our presentation

File: streamlit_dashboard/dashboard.py [hunk 1]
```diff
@@ -15,10 +15,10 @@
 from dashboard_etl import (get_html_from_url, get_website_from_url,
                            scrape_pricing_process, clean_price)
 
+from homepage import home_page
 
-st.set_page_config(
-    layout="wide"
-)
+st.set_page_config(page_title="Price Slashers - Sales Tracker",
+                   page_icon="💸", layout="wide")
```
Comment (indiah444) [inline streamlit_dashboard/dashboard.py:21]: Cool little page config! Love the emoji

File: streamlit_dashboard/logo.png
Comment (indiah444) [inline streamlit_dashboard/logo.png:1]: Love the logo!

**Review by indiah444 — APPROVED** [review]
Love all of this, the homepage looks fantastic, thank you for working so hard on this!

File: streamlit_dashboard/logo.png
Comment (jk-george) [reply streamlit_dashboard/logo.png:1]: Too bad we can't use it :(

**Review by anitamegarry — APPROVED** [review]

**Pull request merged**